dev-gap/polenta/polenta-1.3.10.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gap-pkg
+
+DESCRIPTION="Polycyclic presentations for matrix groups"
+SLOT="0"
+SRC_URI="https://github.com/gap-packages/${PN}/releases/download/v${PV}/${P}.tar.bz2"
+
+LICENSE="GPL-2"
```

Comment:
i think this should be `GPL-2+`

Reply:
Just wondering, are these ebuilds managed manually or by a script? Because all the package metadata is available for each GAP recent GAP release, e.g. https://github.com/gap-system/gap/releases/download/v4.12.2/package-infos.json.gz for GAP 4.12.2 -- so one could parse that and then map the (small) list of SPDX ids used in there to license here (and while at it, also extract URLs of the actual package homepages, instead of pointing to the GAP distro pages for them).

(Of course that would be quite a bit of work, don't feel compelled to do it, just though I should mention it)

Reply:
Done.

These were all written manually. In most (but apparently not all) cases I double-checked the license, dependency info, and package descriptions. We could probably still use that data as a sanity check, but I simply didn't know about it before now.

Not listing the additional homepages was intentional, since it takes a bit of time, is error prone, and clutters up the user interface. I think most people will be happy to visit the gap-system.org page and then follow links from there. But if not, it's good to know that we can obtain them wholesale.